libs/shared/ui/src/lib/components/avatar/avatar.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import { IconEnum } from '@console/shared/enums'
+import { render } from '__tests__/utils/setup-jest'
+import { screen } from '@testing-library/react'
+import { Chance } from 'chance'
+
+import { Avatar, AvatarProps, AvatarStyle } from './avatar'
+
+const props: AvatarProps = {
+  url: ''
+}
+
+const chance = new Chance()
+
+let container: any;
+
+beforeEach(() => {
+  container = document.createElement('div');
+  document.body.appendChild(container);
+});
+
+describe('Avatar', () => {
+
+  it('should render successfully', () => {
+    const { baseElement } = render(<Avatar {...props} />)
+    expect(baseElement).toBeTruthy()
+  })
+
+  it('should apply the accurate classes', () => {
+    props.style = AvatarStyle.NORMAL
+    props.className = 'some-class-name'
+
+    const rendered = render(<Avatar {...props} />)
+
+    const avatar = rendered.container.querySelector('div')
```

Comment:
Try to avoid `querySelector` when possible and prefer `role` or `label` selector as they are less opened to any dom change